src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  onAddMovie: (newMovie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ onAddMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie>();
+  const [hasLoadingError, setHasLoadingError] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  async function findMovie() {
+    setHasLoadingError(false);
+    setLoading(true);
+    const newMovie = await getMovie(query);
+
+    if ('Error' in newMovie) {
+      setHasLoadingError(true);
+    } else {
+      const imgUrl = (newMovie.Poster === 'N/A')
+        ? 'https://via.placeholder.com/360x270.png?text=no%20preview'
+        : newMovie.Poster;
+
+      setMovie({
+        title: newMovie.Title,
+        description: newMovie.Plot,
+        imdbId: newMovie.imdbID,
+        imdbUrl: `https://www.imdb.com/title/${newMovie.imdbID}`,
+        imgUrl,
+      });
+    }
+
+    setLoading(false);
+  }
```

Comment:
use try/catch/finally statement here to avoid errors
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch